save-datasett-in-mottak/Dockerfile

```
@@ -0,0 +1,20 @@
+FROM python:3.10.2-slim AS prepare
```

```suggestion
FROM python:3.10.2 AS prepare
```
Ved fremtidige images så kommer jeg til å fjerne `-slim` fra `prepare` steget. Siden det kan være dependencies som trenger f.eks gcc for å bygge wheels, siden det kan ofte ta lang tid før de er tilgjengelig.
Så ved å bruke "non-slim" image, så får vi med bygge verktøy for å bygge wheels

---

Men runtime, altså lengre ned, kan fortsatt være `-slim` image, siden den ikke trenger å bygge wheels fra tid til annen.